package.json

```diff
@@ -1,33 +1,38 @@
 {
   "name": "@fusebit/monaco-jsonforms",
-  "version": "1.0.8",
+  "version": "1.0.9",
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
   "devDependencies": {
+    "@material-ui/core": "^4.12.4",
     "@rollup/plugin-commonjs": "^22.0.1",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^13.3.0",
+    "@rollup/plugin-replace": "^4.0.0",
     "@rollup/plugin-typescript": "^8.3.4",
+    "@types/material-ui": "^0.21.12",
     "@types/node": "^18.6.3",
     "@types/react": "^18.0.15",
     "@types/styled-components": "^5.1.25",
+    "prettier": "2.0.5",
     "rollup": "^2.77.2",
     "rollup-plugin-dts": "^4.2.2",
     "rollup-plugin-peer-deps-external": "^2.2.4",
     "rollup-plugin-terser": "^7.0.2",
-    "@rollup/plugin-replace": "^4.0.0",
     "tslib": "^2.4.0",
     "typescript": "^4.7.4"
   },
   "peerDependencies": {
-    "react": "16.14.0",
     "@jsonforms/core": "^2.5.2",
+    "@jsonforms/material-renderers": "^2.5.2",
     "@jsonforms/react": "^2.5.2",
-    "@jsonforms/material-renderers": "^2.5.2"
+    "@material-ui/core": "^4.11.2",
+    "react": "^16.14.0"
```

Comment:
Concerns to vet:
  * Changing the react version from `16.14.0` to `^16.14.0`, and how that intersects with the hard-coded versions in the `.html` files.
  * Addition of `@material-ui/core` - is that correctly included, with the right version, in the `.html` files already?